slash_commands/nft.js

```diff
@@ -0,0 +1,75 @@
+const { SlashCommandBuilder } = require('discord.js');
+const { red, cyber } = require('../jsons/colors.json');
+const bHelper = require("../helpers/bloons-general.js");
+
+const builder = new SlashCommandBuilder()
+.setName('nft')
+.setDescription('Calculate Quincy Action Figure buy/sell price in Shol Geraldo\'s shop')
+.addIntegerOption(option => option.setName('unlock_round').setDescription("Start of round when the action figure is unlocked").setRequired(true).setMinValue(1))
```

Comment:
Would it be more straightforward to say "The round Geraldo is placed"?

Reply:
I believe that that's not necessarily the same thing if MK is not on

Reply:
Ah ok